core/primitives/src/views.rs

```diff
@@ -1474,13 +1474,25 @@ pub type StateChangesKindsView = Vec<StateChangeKindView>;
 pub enum StateChangeCauseView {
     NotWritableToDisk,
     InitialState,
-    TransactionProcessing { tx_hash: CryptoHash },
-    ActionReceiptProcessingStarted { receipt_hash: CryptoHash },
-    ActionReceiptGasReward { receipt_hash: CryptoHash },
-    ReceiptProcessing { receipt_hash: CryptoHash },
-    PostponedReceipt { receipt_hash: CryptoHash },
+    TransactionProcessing {
+        tx_hash: CryptoHash,
+    },
+    ActionReceiptProcessingStarted {
+        receipt_hash: CryptoHash,
+    },
+    ActionReceiptGasReward {
+        receipt_hash: CryptoHash,
+    },
+    ReceiptProcessing {
+        receipt_hash: CryptoHash,
+    },
+    PostponedReceipt {
+        receipt_hash: CryptoHash,
+    },
     UpdatedDelayedReceipts,
     ValidatorAccountsUpdate,
+    #[cfg(feature = "ganache")]
+    PatchState,
```

Comment:
I don't think we need this here as I cannot see a use case where you need to query the causes of state changes when you run neard with `ganache`.

Reply:
This was for the sake of keep state change reason realistic, but yeah user run test on ganache won't really query reason for patch_state so we don't need this